Makefile

```diff
@@ -64,6 +64,7 @@ check: python-env
 	${FIND} -name *.py -exec autopep8 -d --max-line-length 120  {} \; | (! grep . -q) || (echo "Code differs from autopep8's style" && false)
 	# Validate that all updates were committed
 	$(MAKE) update
+	$(MAKE) notice
```

Comment:
Yes, and as it only runs on the global `make update` which already does a lot of stuff, it I think it will not make a difference in speed.